lerna.json

```diff
@@ -10,7 +9,8 @@
       "Tag: Internal": ":house: Internal"
     }
   },
-  "version": "9.2.10",
   "npmClient": "yarn",
-  "useWorkspaces": true
+  "useWorkspaces": true,
+  "registry": "https://registry.npmjs.org",
```

Comment:
Do we need this registry?

Reply:
Yes, because yarn defaults to `yarnpkg` when publishing.

Reply:
Even though we are using `npm` when publishing, it's better safe to have this.